src/System.Runtime/src/System/LazyOfTTMetadata.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace System
 {
-    [Serializable]
```

Comment:
Lazy's delegate is marked NonSerialized -- it just initializes Value when serializing.